var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -17,21 +17,43 @@ class PyJaxlib(PythonPackage, CudaPackage):
     url = "https://github.com/google/jax/archive/refs/tags/jaxlib-v0.1.74.tar.gz"
 
     tmp_path = ""
+    buildtmp = ""
 
+    version("0.3.22", sha256="680a6f5265ba26d5515617a95ae47244005366f879a5c321782fde60f34e6d0d")
     version("0.1.74", sha256="bbc78c7a4927012dcb1b7cd135c7521f782d7dad516a2401b56d3190f81afe35")
 
+    # see jaxlib/setup.py for dependencies
     depends_on("python@3.7:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
-    depends_on("py-numpy@1.18:", type=("build", "run"))
-    depends_on("py-scipy", type=("build", "run"))
+
+    depends_on("py-numpy@1.18:", type=("build", "run"), when="@0.1.74")
+    depends_on("py-numpy@1.20:", type=("build", "run"), when="@0.3.22")
+    depends_on("py-scipy@1.5:", type=("build", "run"))
     depends_on("py-absl-py", type=("build", "run"))
-    depends_on("py-flatbuffers@1.12:2", type=("build", "run"))
-    # Bazel 5 not yet supported: https://github.com/google/jax/issues/8440
```

Comment:
It'd be good to keep this comment since the upper bound isn't documented in older deps